charts/wildfly/templates/ingress.yaml

```diff
@@ -0,0 +1,3 @@
+{{- if (and (not (and .Values.deploy.route.enabled (.Capabilities.APIVersions.Has "route.openshift.io/v1/Route"))) .Values.deploy.ingress.enabled) }}
```

Comment:
we should explain somewhere in the README this behaviour so that user would know which resources (route, ingress) will be created based on their target platform (k8s, openshift) and the helm chart configuration